app/src/test/java/org/simple/clinic/editpatient/PatientEditScreenControllerTest.kt

```diff
@@ -656,6 +656,21 @@ class PatientEditScreenControllerTest {
             createExpectedPhoneNumber = { patientId, alreadyPresentPhoneNumber ->
               alreadyPresentPhoneNumber?.copy(number = "12345678") ?: PatientMocker.phoneNumber(patientUuid = patientId, number = "12345678")
             }),
+        generateTestData(
+            patientProfile = generatePatientProfile(shouldAddNumber = false),
+            numberValidationResult = VALID,
+            inputEvents = listOf(
+                PatientEditPatientNameTextChanged("Name"),
+                PatientEditDistrictTextChanged("District"),
+                PatientEditColonyOrVillageChanged("Colony"),
+                PatientEditStateTextChanged("State"),
+                PatientEditGenderChanged(MALE),
+                PatientEditPhoneNumberTextChanged("")),
+            shouldSavePatient = true,
+            createExpectedPatient =
+            { it.copy(fullName = "Name", gender = MALE) },
```

Comment:
Name and values on different lines looks weird. Can we do single line instead?

```kotlin
createExpectedPatient = { it.copy(fullName = "Name", gender = MALE) },
```

Reply:
Actually, feel free to do this in your next PR. This isn't blocking this fix.

Reply:
Yup, I'll make this change.